src/types.jl

```diff
@@ -56,15 +53,12 @@ end
 mutable struct Extrema
     emin::Float64
     emax::Float64
-end
-Extrema() = Extrema(Inf, -Inf)
 
-# -----------------------------------------------------------
+    Extrema() = new(Inf, -Inf)
```

Comment:
Did you really meant to override the default constructor here?
That results in `Extrema(1,2)` to throw a `MethodError`

Reply:
It see it now in the logs with the `hdf5` backend. And of course these test were so badly written that they only warn and don't throw errors. I will fix this (and enhance the tests).